clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/handlers/LoadBalancerV2UpsertHandler.groovy

```diff
@@ -34,18 +34,61 @@ class LoadBalancerV2UpsertHandler {
   private static final String ATTRIBUTE_IDLE_TIMEOUT = "idle_timeout.timeout_seconds"
   private static final String ATTRIBUTE_DELETION_PROTECTION = "deletion_protection.enabled"
 
+  //Defaults for Target Group Attributes
+  private static final String DE_REGISTRATION_DELAY = "300"
+  private static final Boolean STICKINESS_ENABLED = false
+  private static final String STICKINESS_TYPE = "lb_cookie"
+  private static final String STICKINESS_DURATION = "86400"
+  private static final Boolean PROXY_PROTOCOL_V2 = false
+  /** The following attribute is supported only if the target is a Lambda function. */
+  private static final Boolean MULTI_VALUE_HEADERS_ENABLED = false
+
   private static Task getTask() {
     TaskRepository.threadLocalTask.get()
   }
 
-  private static String modifyTargetGroupAttributes(AmazonElasticLoadBalancing loadBalancing, LoadBalancer loadBalancer, TargetGroup targetGroup, UpsertAmazonLoadBalancerV2Description.Attributes attributes) {
-    return modifyTargetGroupAttributes(loadBalancing, loadBalancer, targetGroup, attributes, null)
+  //Create Target Group Attributes with values provided in description, set to defaults other wise
+  static String createTargetGroupAttributes(AmazonElasticLoadBalancing loadBalancing, LoadBalancer loadBalancer, TargetGroup targetGroup, UpsertAmazonLoadBalancerV2Description.Attributes attributes, DeployDefaults deployDefaults) {
+    def targetGroupAttributes = []
+    log.info("Creating target group attributes for targetGroup {} with attributes {} ", targetGroup.targetGroupName, attributes)
```

Comment:
This won't pretty-print `attributes` if it doesn't have a `toString()` on it or some lombok annotation.

Reply:
👍 fixed